fastlane/Fastfile

```diff
@@ -176,7 +176,7 @@ platform :android do
       paths_of_files_to_update: [constants_path]
     )
     gradle(
-      task: ':common:test',
+      task: ':common:testIntegrationTestDebugUnitTest',
```

Comment:
Do you think it's worth it to add a comment on why we are running that particular flavor?

Reply:
Yeah absolutely, will add